### ISSUE TITLE
Cannot Run Server - The 1.7.10 Pack Server 0.9.4

### ISSUE THREAD

**Issue by syephro** [xJon/The-1.7.10-Pack#1298]
http://pastebin.com/tXrjDZrU - That's the errors I get when I run "LaunchServer"

Nothing has been edited. I added a minimap mod, as there wasn't one. But it did this stuff before I added the mod so.

Everything works in single player, though.

There's a lot more that comes, but the command prompt window won't let me go any farther up. So, that's all I can copy and paste. 

The most common issue I see that wasn't able to get pasted was "Does not have an MCVersion annotation" or something like that.

http://pastebin.com/y9KbWyLn - There's the crash report.

Basically just took the server download from your site and tried to run it. Never got anywhere.

**Comment by xJon** [comment]
Why did you add a minimap mod?
The 1.7.10 Pack already has one, and it needs to be installed only on client side.

**Comment by syephro** [comment]
I added one because there wasn't one. I checked. Nowhere in the mod folder from what downloaded through the Technic launcher was there a minimap mod. (I'm not running this through the technic launcher, I'm running it through Forge and the standard MC launcher). And, that still doesn't answer my question. It didn't work before the minimap mod, and it doesn't work after. So, the mod I added had nothing to do with the crash/not loading issue.
